src/normalizeEntityUri.ts

```diff
@@ -55,7 +55,34 @@ function normalizeEntityUri (uriOrEntity: string | ResourceInterface | null = ''
  */
 function normalizeUri (uri: unknown, baseUrl: string): string | null {
   if (typeof uri !== 'string') return null
-  return sortQueryParams(uri).replace(new RegExp(`^${baseUrl}`), '')
+  const sorted = sortQueryParams(uri)
+  const simpleReplace = sorted.replace(new RegExp(`^${baseUrl}`), '')
+  if (baseUrl && baseUrl.length > 2 && simpleReplace === uri) {
```

Comment:
Why `length > 2`? Is this to avoid trailing slashes? Then shouldn't it be `length >= 2`?

Reply:
I removed it. It was defensive programming, but the tests show that this is not necessary.